pages/linux/cmus.md

```diff
@@ -0,0 +1,24 @@
+# cmus
+
+> Commandline C* Music Player.
+> Arrow keys to navogate, `<enter/return>` to select, numbers 1-8 switch between different views.
+
+- Open cmus from specified directory:
+
+`cmus {{path/to/directory}}`
+
+- Add file/directory to library:
+
+`:add {{/path/to/file_or_directory}}`
+
+- Pause/unpause current song:
+
+`c`
```

Comment:
Ok. It would be nice if we could come up with a nice mnemonic for it, even if it isn't the original meaning (if any). But that would be just a bonus :) it definitely won't hold up merging.